src/components/accordion/AccordionItem.tsx

```diff
@@ -16,9 +18,9 @@ function AccordionItem({
     "after:content-[''] after:shrink-0 after:w-[14px] after:h-[14px] after:ml-auto after:transition-[transform] after:duration-[200ms] after:ease-out after:bg-[length:14px_14px] after:bg-no-repeat after:bg-arrow"
   return (
     <li className="border-0 ">
-      <h2 className="m-0 w-full">
-        <button
-          className={clsx("b-0 flex w-full cursor-pointer items-center ", afterStyle, {
+      <h2 className="m-0 w-full hover:opacity-70">
+        <div
+          className={clsx("b-0 flex w-full cursor-pointer items-center" + afterStyle, {
```

Comment:
Use a comma instead, as clsx automatically joins classes.

Reply:
This is dawid stuff, idk how clsx works or what this does